pkg/urbit/jets/tree.c

```diff
@@ -455,6 +451,19 @@ static c3_c* _141_pen_loot_ha[] = {
     0
   };
   static u3j_harm _141_pen__ut_nest_a[] = {{".2", u3wfu_nest}, {}};
+      static u3j_harm _141_pen__ut_nest_dext_a[] = {{".2", u3wfu_nest_dext}, {}};
+    static u3j_core _141_pen__ut_nest_in_d[] =
+      {
+        // TODO: compute hash
```

Comment:
The `TODO`'s aren't necessary, but please do add the `static c3_c* _141_pen__ut_nest...ha[]` declarations, initialized to `{0}`.